app/services/file_manager/file_download.py

```diff
@@ -80,6 +84,7 @@ def prepare_download(self):
             files.append({'id': f})
         payload = {
             'files': files,
+            'zone': self.zone,
```

Comment:
Same as above, this self.zone haven't seen it initiated

Reply:
will update it